### ISSUE TITLE
Support to use the source port with dynamic spoofed IP + port

### ISSUE THREAD

**Issue by LionsAd** [sonicsnes/udp-broadcast-relay-redux#3]
```
@@ -515,6 +520,9 @@ int main(int argc,char **argv) {
                        if (spoof_addr == inet_addr("1.1.1.1")) {
                                fromAddress = iface->ifaddr;
                                fromPort = port;
+                       } else if (spoof_addr == inet_addr("2.2.2.2")) {
+                               fromAddress = iface->ifaddr;
+                               fromPort = origFromPort;
                        } else if (spoof_addr) {
                                fromAddress.s_addr = spoof_addr;
                                fromPort = origFromPort;
```

it would be great to support to use origFromPort even with the dynamic spoofed IP, based on the interface address.

**Comment by sonicsnes** [comment]
Works for me. What protocol is this needed for?

**Comment by LionsAd** [comment]
Nintendo Switch - Local lan play - seems to get confused if source ports are wrong - not yet 100% confirmed though.

**Comment by sonicsnes** [comment]
Merged, but used 1.1.1.2.
